src/components/Layout/MainHeader/MainHeaderStyles.ts

```diff
@@ -22,11 +22,42 @@ export const StyledMainHeader = styled.header`
 
 export const MainHeadingBrandName = styled.div`
   position: absolute;
+  text-align: center;
   left: 50%;
   transform: translateX(-50%);
   font-size: 3.5rem;
   font-weight: normal;
   font-family: "Rubik Puddles", cursive;
   word-spacing: 0.5rem;
-  min-width: 400px;
+
+  @media (max-width: 540px) {
```

Comment:
it's better when all the media queries are consistent and coming from your theme